ui/src/components/layout/flex.styled.tsx

```diff
@@ -0,0 +1,10 @@
+import { dripStitches } from '../../theme/stitches'; //TODO replace with absolute path

```

Comment:
There are files with suffix `.styled.tsx` and `.styles.tsx` lets keep using a single one for all of them.

Reply:
agree! I would do it